src/Qir/Tests/OpenSystemsSimulator/OpenSystemsSimulatorTests.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <complex>
+#include <memory>
+#include <vector>
+
+#define CATCH_CONFIG_MAIN // This tells Catch to provide a main() - only do this in one cpp file
+#include "catch.hpp"
+
+#include "QuantumApi_I.hpp"
+#include "SimFactory.hpp"
+#include "context.hpp"
+
+using namespace Microsoft::Quantum;
+using namespace std;
+
+TEST_CASE("Open systems simulator: X and measure", "[open_simulator]")
```

Comment:
Please make sure you cover all of the implemented functionality.

Reply:
Yes, absolutely. These tests are for the QIR runtime integration, which won't be supported in the initial version, given focus on usage from Python host programs for now.